Index: src/Lesson4_5/ModuloExamples.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Lesson4_5/ModuloExamples.java b/src/lesson_04_5/ModuloExamples.java
rename from src/Lesson4_5/ModuloExamples.java
rename to src/lesson_04_5/ModuloExamples.java
--- a/src/Lesson4_5/ModuloExamples.java	
+++ b/src/lesson_04_5/ModuloExamples.java	
@@ -1,4 +1,4 @@
-package Lesson4_5;
+package lesson_04_5;
 
 public class ModuloExamples {
         public static void main(String[] args) {
